projects/arlas-toolkit/src/lib/services/startup/startup.service.ts

```diff
@@ -414,55 +441,104 @@ export class ArlasStartupService {
         const usePersistence = (!!settings && !!settings.persistence && !!settings.persistence.url
           && settings.persistence.url !== '' && settings.persistence.url !== NOT_CONFIGURED);
-        if (usePersistence) {
-          // To open reconnect dialog on silent refresh failed
-          authService.silentRefreshErrorSubject.subscribe(error =>
-            // eslint-disable-next-line arrow-body-style
-            this.persistenceService.list('config.json', 10, 1, 'asc').subscribe(data => {
-              return;
-            }));
-        }
-        this.fetchInterceptorService.applyInterceptor();
-        authService.canActivateProtectedRoutes.subscribe(isActivable => {
-          if (isActivable) {
-            // ARLAS-persistence
-            this.persistenceService.setOptions({
-              headers: {
-                Authorization: 'bearer ' + authService.accessToken
-              }
-            });
-            // ARLAS-server
-            this.fetchOptions.headers = {
-              Authorization: 'bearer ' + authService.accessToken
-            };
-            // ARLAS-Permission
-            this.persmissionService.setOptions({
-              headers: {
-                Authorization: 'bearer ' + authService.accessToken
-              }
-            });
-          } else {
-            this.persistenceService.setOptions(this.getOptions());
-            this.persmissionService.setOptions(this.getOptions());
+
+        if (useAuthentOpenID) {
+          const authService: AuthentificationService = this.injector.get('AuthentificationService')[0];
+          if (usePersistence) {
+            // To open reconnect dialog on silent refresh failed
+            authService.silentRefreshErrorSubject.subscribe(error =>
+              // eslint-disable-next-line arrow-body-style
+              this.persistenceService.list('config.json', 10, 1, 'asc').subscribe(data => {
+                return;
+              }));
           }
-          this.collaborativesearchService.setFetchOptions(this.fetchOptions);
-          resolve(settings);
-        });
+          this.fetchInterceptorService.applyInterceptor();
+          authService.canActivateProtectedRoutes.subscribe(isActivable => {
+            if (isActivable) {
+              // ARLAS-persistence
+              this.persistenceService.setOptions({
+                headers: {
+                  Authorization: 'bearer ' + authService.accessToken
+                }
+              });
+              this.permissionService.setOptions({
+                headers: {
+                  Authorization: 'bearer ' + authService.accessToken
+                }
+              });
+              // ARLAS-server
+              this.fetchOptions.headers = {
+                Authorization: 'bearer ' + authService.accessToken
+              };
+              // ARLAS-Permission
+              this.permissionService.setOptions({
```

Comment:
There is a duplicate of the setOptions for the permissionService

Reply:
fixed